R/bcdc_options.R

```diff
@@ -140,7 +140,7 @@ bcdc_get_capabilities <- function() {
 bcdc_get_wfs_records <- function() {
   doc <- bcdc_get_capabilities()
 
-  if (is.null(doc)) stop("Unable to access wfs record listing", call. = FALSE)
+  if (is.null(doc)) stop("Unable to access wfs listing from server", call. = FALSE)
```

Comment:
I wonder how we could make this more actionable. For example could we direct users to this link? https://uptime.com/statuspage/bcgov-dss/1389727

Or we could do something like:
```suggestion
  if (is.null(doc)) stop("Unable to access wfs listing from server. Please open an issue. ", call. = FALSE)
```

Reply:
Yeah, are we sure that's the correct link for the whole wfs service? Or is that for a particular feature (most of the other ones are, for wildfire etc?)

Reply:
Going to take your suggestion for now and see about where/how to integrate the uptime page...